tap_mysql/sync_strategies/binlog.py

```diff
@@ -318,7 +314,8 @@ def handle_delete_rows_event(event, catalog_entry, state, columns, rows_saved, t
     db_column_types = get_db_column_types(event)
 
     for row in event.rows:
-        event_ts = datetime.datetime.utcfromtimestamp(event.timestamp).replace(tzinfo=pytz.UTC)
+        event_ts = datetime.datetime.utcfromtimestamp(event.timestamp) \
+            .replace(tzinfo=pytz.UTC).isoformat()
```

Comment:
what does the extra `.isoformat()` do?

Reply:
So if we keep it as is, the code [here](https://github.com/transferwise/pipelinewise-tap-mysql/blob/bec15aa3bab049de482323c84cc732fc3f11aaf5/tap_mysql/sync_strategies/binlog.py#L143) ends up creating datetime string with double `+00:00` like so: `2021-03-31T12:10:30+00:00+00:00`

Reply:
oh interesting. Why it's not causing issues at the moment? Or it does and it's actually a bugfix?

Reply:
I don't why but most `_sdc_deleted_at` columns are varchar in SF, so even if tap sends a string datetime formatted badly, SF won't complain and will gladly take it.

I'm trying to find a table in Prod that has a not null `_sdc_deleted_at` but can't find any yet, probably cuz we have hard delete enabled.

Reply:
So basically, it's a logical bug that doesn't break anything.

Reply:
🌶️  <- pepper instead of hot potato

Reply:
but that seems like another question not related to this PR. thanks for clarifying 🙇

Reply:
What's more is that we have this [function](https://github.com/transferwise/pipelinewise-tap-mysql/blob/bec15aa3bab049de482323c84cc732fc3f11aaf5/tap_mysql/sync_strategies/binlog.py#L46) that adds the sdc column with format but don't know what's happening that it ends up being varchar in SF